src/validation-error-message-formatter.js

```diff
@@ -0,0 +1,401 @@
+// ATTENTION: Please keep these entries organized in alphabetical order
```

Comment:
The contents of this file are identical to the previous revision of `etc/validation-error-message-formatter.js`.